src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -1053,60 +1073,20 @@ public static boolean backupColonyData()
         return true;
     }
 
-    public static void addToZipFile(final String fileName, final ZipOutputStream zos, final File folder)
-    {
-        final File file = new File(folder, fileName);
-        try(FileInputStream fis = new FileInputStream(file))
-        {
-            zos.putNextEntry(new ZipEntry(fileName));
-            Files.copy(file, zos);
-            fis.close();
-        }
-        catch (final Exception e)
-        {
-            /**
-             * Intentionally not being thrown.
-             */
-            Log.getLogger().warn("Error packing " + fileName + " into the zip.");
-        }
-    }
-
-    /**
-     * Load a file and return the data as an NBTTagCompound.
-     *
-     * @param file The path to the file.
-     * @return the data from the file as an NBTTagCompound, or null.
-     */
-    private static NBTTagCompound loadNBTFromPath(@Nullable final File file)
-    {
-        try
-        {
-            if (file != null && file.exists())
-            {
-                return CompressedStreamTools.read(file);
-            }
-        }
-        catch (final IOException exception)
-        {
-            Log.getLogger().error("Exception when loading ColonyManger", exception);
-        }
-        return null;
-    }
-
     /**
      * Read Colonies from saved NBT data.
      *
      * @param compound NBT Tag.
      */
     public static void readFromNBT(@NotNull final NBTTagCompound compound, @NotNull final World world)
     {
-        if(!compound.hasKey(TAG_DISTANCE))
+        if (!compound.hasKey(TAG_DISTANCE))
         {
             Configurations.gameplay.workingRangeTownHallChunks =
-                    (int) ((Math.cos(45.0 / HALF_A_CIRCLE * Math.PI) * Configurations.gameplay.workingRangeTownHall) / BLOCKS_PER_CHUNK);
+              (int) ((Math.cos(45.0 / HALF_A_CIRCLE * Math.PI) * Configurations.gameplay.workingRangeTownHall) / BLOCKS_PER_CHUNK);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 45.0 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)